package/utils/get_style_rule.js

```diff
@@ -14,7 +15,7 @@ const getStyleRule = (test, modules = false, preprocessors = []) => {
         sourceMap: true,
         importLoaders: 2,
         modules: modules ? {
-          localIdentName: '[name]__[local]___[hash:base64:5]'
+          localIdentName: isProduction ? '[hash:base64]' : '[path][name]__[local]'
```

Comment:
The path of the resource is more useful than a hash in development.

https://github.com/webpack/loader-utils#interpolatename

```
[path] the path of the resource relative to the context query parameter or option.
```